component/libfuse/libfuse_handler.go

```diff
@@ -346,8 +346,17 @@ func libfuse_destroy(data unsafe.Pointer) {
 }
 
 func (lf *Libfuse) fillStat(attr *internal.ObjAttr, stbuf *C.stat_t) {
-	(*stbuf).st_uid = C.uint(lf.ownerUID)
-	(*stbuf).st_gid = C.uint(lf.ownerGID)
+	if attr.UID != nil {
```

Comment:
ok for this kind of check may be either *string or *int is better but we might be reading from storage less often and returning it back to kernel more often so storing as *int will be a better choice.